tests/Jackalope/Transport/DoctrineDBAL/CachedClientTest.php

```diff
@@ -7,9 +7,13 @@
 
 class CachedClientTest extends FunctionalTestCase
 {
+    /** @var \PHPUnit_Framework_MockObject_MockObject */
+    private $cacheMock;
+
     protected function getClient(Connection $conn)
     {
-        return new CachedClient(new \Jackalope\Factory(), $conn);
+        $this->cacheMock = $this->getMock('\Doctrine\Common\Cache\MemcachedCache');
```

Comment:
it seems this class is not always available on travis-ci. can we mock the interface instead of a concrete implementation? or base the mock on the ArrayCache that is the default cache used in the client if none is specified.